spec/index.bs

```diff
@@ -6361,6 +6361,12 @@ and <dfn dfn>vertex data type</dfn> for the component.
   * `sint` = signed int
   * `float` = floating point
 
+The multi-component formats specify the number of components after "x".
+As such, {{GPUVertexFormat/"sint32x3"}} denotes a 3-component vector of `i32` values in the shader.
```

Comment:
This particular example creates a false equivalency between "sint32" and "i32", when in fact "sint16" also maps to i32 (as seen below). Maybe skip talking about WGSL until later:

```
As such, {{GPUVertexFormat/"sint16x2"}} denotes a 2-component vector of 16-bit signed integer values.
```